backend/src/main/java/group/app/backend/controller/AppController.java

```diff
@@ -0,0 +1,20 @@
+package group.app.backend.controller;
+
+import group.app.backend.auth.AuthService;
+import group.app.backend.auth.JsosAuthProperties;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+public class AppController {
+
+    private final AuthService authService;
```

Comment:
Shouldn't we inject this dependency?

Reply:
Spring does it for us by @RequiredArgsConstructor